src/main/java/org/folio/dcb/client/feign/config/decoder/DcbErrorDecoder.java

```diff
@@ -0,0 +1,34 @@
+package org.folio.dcb.client.feign.config.decoder;
+
+import feign.codec.ErrorDecoder;
+import lombok.extern.log4j.Log4j2;
+import org.folio.dcb.exception.ResourceAlreadyExistException;
+import org.springframework.http.HttpStatus;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+@Log4j2
+public class DcbErrorDecoder implements ErrorDecoder {
```

Comment:
I have left a comment on ServicePointServiceImpl class. If we follow that, we don't need to have this class as well as DcbClientConfiguration class. Please check